### DISCUSSION TITLE
How to get a new badge on github?

### DISCUSSION THREAD

**Discussion by whalescats** [community/community#18476] [Other Feature Feedback, Questions, & Ideas] [answered]
How to get a new badge on github?

**Comment by HUMBLEGROUND** [comment, marked as answer]
Maybe this can help https://github.com/Schweinepriester/github-profile-achievements

**Reply by lucasfrag** [reply]
Thanks!!!

**Reply by gabivechiatto** [reply]
Nice!!

**Reply by assebc** [reply]
https://github.com/github-community/community/discussions/18294

**Reply by ibnuuby** [reply]
thanks

**Reply by MiaCutie** [reply]
thanks

**Reply by siakamansaly** [reply]
Thanks!

**Reply by MarcelaDJ** [reply]
Thank you for the info

**Reply by ghost** [reply]
ty

**Reply by Rems08** [reply]
Nice !

**Reply by paradox-bd** [reply]
yeah

**Reply by firmnardians** [reply]
Thanks

**Reply by ehmedP** [reply]
Thanks

**Reply by rapha661** [reply]
Thanks

**Reply by rmarye90** [reply]
thanks

**Reply by wildanarkan** [reply]
thanks

**Reply by tomasoliveira21** [reply]
thanks!

**Comment by weishaodaren** [comment]
After answering two discussions, You will get Galaxy Brain badge

**Reply by Seun0001** [reply]
That's lovely then.

**Reply by Rems08** [reply]
For sure

**Reply by paradox-bd** [reply]
sure

**Reply by ThomasTannenberg** [reply]
sure

**Comment by AldonahZero** [comment]


**Reply by RaphaelAlcantara** [reply]
yes, is very cool

**Reply by MiaCutie** [reply]
loloololol

**Reply by NikolaRoganovic** [reply]
cool

**Reply by Hikkywannafly** [reply]
cool

**Comment by ZenithClown** [comment]
Pull Shark Badge by merging PRs. Own repository works. Galaxy brain badge by answering two discussions.

**Reply by RaphaelAlcantara** [reply]
Tks

**Reply by Youaymo** [reply]
Very good

**Reply by Seun0001** [reply]
Oh! Wow, that's cool 🆒

**Reply by NikolaRoganovic** [reply]
Cool!

**Comment by ryudongjae** [comment]
Pull Shark Badge by merging PRs. Own repository works. Galaxy brain badge by answering two discussions.

**Comment by neriidesu** [comment]
Pull Shark Badge by merging PRs. Own repository works. Galaxy brain badge by answering two discussions
You also have one for making a PR and getting it accepted without it getting reviewed.

**Reply by Youaymo** [reply]
What??

**Comment by ethanxv24** [comment]
Pull Shark Badge by merging PRs. Own repository works. Galaxy brain badge by answering two discussions.

**Reply by gabivechiatto** [reply]
Cool

**Comment by themiralay** [comment]
for Galaxy badge: Answer two discussions
for Pull Shark badge : Have two pull requests merged
for yolo badge : Merge code without review
for more info : https://github.com/Schweinepriester/github-profile-achievements

**Comment by AnuraghSarkar** [comment]
for Galaxy badge: Answer two discussions
for Pull Shark badge : Have two pull requests merged
for yolo badge : Merge code without review
for more info : https://github.com/Schweinepriester/github-profile-achievements
This may help you a bit 
#ctto

**Comment by ryudongjae** [comment]
for Galaxy badge: Answer two discussions
for Pull Shark badge : Have two pull requests merged

**Reply by AnuraghSarkar** [reply]
not just answering will help it should be marked too

**Comment by Youaymo** [comment]
It's didint work

**Comment by opensea0508** [comment]
Quickdraw badge: Closed an issue or a pull request within 5 min of opening
Pair Extraordinaire badge: Coauthored  in a merged pull request

**Comment by Ismoh** [comment]
I love to use [shields.io](https://shields.io/) or badgen, etc pp.
There are default badges, examples and you can create custom static badges!
I hope this helps!

**Reply by Ismoh** [reply]
Just realized you're talking about achievements not badges :D

**Comment by lechithinh** [comment]
Galaxy Brain: Two accepted answers
Pull Shark: 2 pull requests merged
YOLO: Merged a pull request without code review
Quickdraw: Closed an issue or a pull request within 5 min of opening

**Comment by Hikkywannafly** [comment]
for Galaxy badge: Answer two discussions
for Pull Shark badge : Have two pull requests merged
for yolo badge : Merge code without review
for more info : https://github.com/Schweinepriester/github-profile-achievements

**Reply by Hikkywannafly** [reply]
cool

**Comment by AneeVerse** [comment]
I got a nice documentation check it out
https://github.com/Schweinepriester/github-profile-achievements

**Comment by Maham-j** [comment]
This can help https://github.com/Schweinepriester/github-profile-achievements

**Comment by talha4real** [comment]
answer two discussions you will get a galaxy brain badge

**Reply by hasanshahriar32** [reply]
Already tried... 🙂
The answer must me acceptable

**Comment by mdtanvirahamedshanto** [comment]
two pull request get one .

**Comment by firmnardians** [comment]
You can read on this link https://github.com/Schweinepriester/github-profile-achievements

**Comment by JunLovin** [comment]
Maybe this can help you https://github.com/Thinkright20/Profile-Badges

**Comment by hasanshahriar32** [comment]
Check this repo.. they have explained throughly

[https://github.com/Schweinepriester/github-profile-achievements]()

**Comment by github-staff** [comment]
Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀

**Comment by ghost** [comment]
you should mark this reply as the answer so that i can get a badge ;) pleaseeeeeeeeee

**Reply by Sharon-SHH** [reply]
I also found [this]( https://github.com/4xmen/Get-Github-Achievements/tree/main). It's helpful and simple to show how to get badges.

**Comment by Chamindu77** [comment]
After answering two discussions, You will get Galaxy Brain badge
yolo badge will get merge code without review

**Comment by deconasser** [comment]
refer to: https://www.youtube.com/watch?v=oJWFfp3sGYA&t=619s

**Comment by 3bHussein** [comment]
Already tried...

**Comment by Hotanya** [comment]
This has already been answered but here is another link which gives good information about GitHub badges - https://githubachievements.com/

**Comment by Revoluti0n** [comment]
Pair extraordinaire:

1. Create a repo;

2. Go to this repo settings, on the left options access the "Collaborators" then add some account; (https://github.com/{your_username}/{your_repo_name}/settings/access)

3. After the invitation for collab accepted, just create some branch on this repo, make some changes and push it:
git checkout -b {branch_name}

...do some changes...

git add .
git commit -m "Some message


Co-authored-by: {username} <{user_email}>
Co-authored-by: {collab_username} <{collab_user_email}>"
git  push origin {branch_name}

4. Go to the repository, create a PR from the created branch then merge it.

Thats it.
